src/button/ReactionToggleButton.module.css

```
@@ -46,25 +48,23 @@
   border-radius: var(--cpd-radius-pill-effect);
 }
 
+@media (max-width: 800px) {
+  .reactionPopupMenuModal .reactionButton {
+    padding: 1em;
+    font-size: 1em;
+    width: 1em;
+    height: 1em;
+    min-block-size: unset;
```

```suggestion
    min-block-size: unset;
    border: none;
```
To match the designs.